sovrin_common/constants.py

```diff
@@ -1,6 +1,94 @@
 from typing import NamedTuple
 
+from plenum.common.constants import TXN_TYPE, TARGET_NYM, ORIGIN, DATA, RAW, ENC, HASH, NAME, VERSION, TYPE, \
+    POOL_TXN_TYPES, ALIAS, \
+    VERKEY
+from sovrin_common.roles import Roles
+from sovrin_common.transactions import SovrinTransactions
+
 Environment = NamedTuple("Environment", [
     ("poolLedger", str),
     ("domainLedger", str)
 ])
+
+ROLE = 'role'
```

Comment:
Several places in this file have very verbose code. For example, this block has a bunch of constants, all of which are an upper-case version of their textual value. Then there is a list of allOpKeys, then a block of code that makes a local symbol that's the same as its SovrinTransactions.X.value equivalent, then a definition of IDENTITY_TXN_TYPES = {...}. I am wondering if this code is overly brittle (susceptible to human error with multiple places to be updated/edited by hand. Is there any easy way to make it less so? Perhaps not; this comment isn't a requirement that you do something different--just a note/suggestion.